src/Bicep.Core/LanguageConstants.cs

```diff
@@ -310,5 +314,25 @@ public static TypeSymbol CreateModuleType(IEnumerable<TypeProperty> paramsProper
 
             return new ModuleType(typeName, moduleScope, moduleBody);
         }
+
+        public static TypeSymbol CreateTestType(IEnumerable<TypeProperty> paramsProperties, string typeName)
```

Comment:
[nit/optional]: I can see that you were following the example of `CreateModuleType` here, but I don't think it makes sense for this to be a public method or really part of `LanguageConstants` at all. I would move this to `DeclaredTypeManager` so it could be a private method.